Update resp.go

mlaoji opened the pull request:
show errors when NIL response ,  is very valuable for some type like zset

mlaoji committed:
Update resp.go

show errors when NIL response ,  is very valuable for some type like zset

mediocregopher commented:
Hey @mlaoji, thanks for the PR! A lot of people have been confused by how radix handles nil responses, and this is a great way to solve that problem I think. if you could make `errResNil` into `ErrRespNil` then I'd be happy to merge, it's better to expose the error so people can check it directly (like in `io.EOF` or `context.Canceled`) rather than checking the string.

mlaoji committed:
Update resp.go

Yes! A better way to expose the error so people can check it directly rather than checking the string.
